Add blog section to Signalen website (#58)
Signalen/signalen.org: src/pages/news/02-11-2021-welcome.en.md

---
path: "/en/news/02-11-2021-welcome/"
date: "2021-02-11"
title: "Welcome to the Signalen blog"
lang: en
template: default
---

> ## Welcome to the Signalen blog

Reviewer: Do we want to call it blog or news?

> There are many developments around Signalen. That is why it is sometimes difficult to keep up to date on all developments.

Reviewer: ```suggestion
Signalen sees a lot of development, which means that it is sometimes difficult to keep up to date on all that is happening.
```
